tests/test_exact_matching.py

```diff
@@ -0,0 +1,63 @@
+from mapillm.exact_matching import ExactMatchingTool
+import pytest 
+
+@pytest.fixture
+def exact_match():
+    return ExactMatchingTool()
+
+
+def test_exact_match(exact_match):
+    formula="LiCoO2"
+    space_group = 12
+    lattice="Monoclinic"
+    density=4.7
+    prop="band_gap"
+
+    output = exact_match.query_material_property(formula, prop, one_only=False)
```

Comment:
This has the potential to fail in future as new materials are added to materials project. 
Maybe assert len(output) >= 9 is a better test